README.md

```diff
@@ -213,12 +216,19 @@ kicking, party transfer buttons, etc. Included in the Party Manager GUI.
 You can open it by typing the command ``/pm`` or by using the keybinding.
 Default: ``M``
 
-![Custom Main Menu](images/party_manager.png)
+<img src="images/party_manager.png" alt="Party Manager" width="600" height="350" style="text-align: center;"> 
 
+
+### Custom Themes
+
+This mod supports a variety of different themes, including dark themes, very dark themes, colorful themes, and controversially, light themes. You can also choose your own accent color, and create your own themes in the OneConfig menu under the theme section. These themes only affect the PSS user interface.
+
+                                    
 ### Pearl Refill
 
 Do you use pearls for your Dungeon runs and want to refill them backup to 16? With the Pearlrefill command you can automatically refill it back up to 16 from any other Stacksize, there is also a Keybind (Default P) and an option to enable auto refill at Dungeon start (use at your own risk)
```

Comment:
> Ah sorry forgot that existed, can you add zhe PearlRefill Keybind (Default P)?
@j10a1n15 

are you stoopid

Reply:
i did improve it

Reply:
Why do we have two pearl refill entries

Reply:
<img width="815" alt="image" src="https://github.com/PartlySaneStudios/partly-sane-skies/assets/83100266/aac69544-86b7-4a3e-8454-bd3c44644a9f">

wdym